### PR TITLE
[FEAT] Mutant Crops

### PR DESCRIPTION
**Pull request by adamhannigan**
# Description

This PR includes the boilerplate actions for finding a mutant crop. 

On a harvest reward, detect if there is a mutant crop - This does not guarantee a user a mutant but it does give them a chance of trying to catch it.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Testnet API + Testnet contracts

# TODO:

- [ ] Searching animation
- [ ] Minting action testing

Screen Shot 2022-05-27 at 8 11 58 am
Screen Shot 2022-05-26 at 8 03 58 am
Screen Shot 2022-05-27 at 8 16 08 am

### CODE REVIEW

File: src/lib/blockchain/MutantCrop.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
+/**
+ * A cross chain cross-over!
```
Comment (tiffanydys) [inline src/lib/blockchain/MutantCrop.ts:2]: update comments/description (remove MoM related stuff)

**Review by tiffanydys — COMMENTED** [review]
a few comments 😊

File: src/features/game/types/index.ts [hunk 2]
```diff
@@ -147,6 +147,7 @@ export const KNOWN_IDS: Record<InventoryItemName, number> = {
   "Easter Bunny": 909,
   "Engine Core": 910,
   Observatory: 911,
+  "Mutant Crop": 911,
```
[the remark this answers is not in the document]
Comment (adamhannigan) [reply src/features/game/types/index.ts:150]: Yes, good catch 🤗

File: src/lib/blockchain/MutantCrop.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        .on("error", function (error: any) {
+          console.log({ error });
+
+          reject(parseMetamaskError(error));
+        })
+        .on("transactionHash", function (transactionHash: any) {
+          console.log({ transactionHash });
+        })
+        .on("receipt", function (receipt: any) {
+          console.log({ receipt });
```
Comment (Ghachem) [inline src/lib/blockchain/MutantCrop.ts:86]: console. log

**Comment by MrSnorch** [comment]
IMG_20220623_083911

**Comment by drovishki3000** [comment]
Dear Adam!
We are  very passionate about the game and are also working to develop the community. We understand that there is a lot of work, but maybe there is an option to run a test version of at least one mutant?

**Comment by adamhannigan** [comment]
@drovishki3000 Thanks for the feedback. We will aim to release this in the next 2-3 weeks. It still requires a bit of security testing and auditing. Right now we've just got our hands full with Land expansion and trading

**Comment by Alabis** [comment]
> @drovishki3000 Thanks for the feedback. We will aim to release this in the next 2-3 weeks. It still requires a bit of security testing and auditing. Right now we've just got our hands full with Land expansion and trading

Thank you. We are waiting for this event from the very beginning of the game and hold HOLD. Please contribute to its development. This is the only NFT that has been released for a long time and has no use in the game.

**Pull request closed without merging**